src/LHScriptX/FeatureScriptCommands.cpp

```diff
@@ -295,827 +289,629 @@ inline float GetSize(const float size)
-	float rotation = -params[4].GetFloat();
-	float lean = -params[5].GetFloat();
-	float size = params[6].GetFloat();
-	auto mobileStaticInfo = MobileStaticInfo(type);
 
-	const glm::vec3 pos(position.x, island.GetHeightAt(position) + verticalOffset, position.y);
-	const glm::mat3 rot = glm::eulerAngleXYZ(pitch, rotation, lean);
-	const glm::vec3 scale(size);
+	glm::vec3 offset(0.0f, verticalOffset, 0.0f);
 
-	registry.Assign<MobileStatic>(entity, mobileStaticInfo);
-	registry.Assign<Transform>(entity, pos, rot, scale);
+	registry.Assign<MobileStatic>(entity, MobileStaticInfo(type));
+	registry.Assign<Transform>(entity, position + offset, glm::eulerAngleXYZ(-pitch, -rotation, -lean), glm::vec3(scale));
 }
 
-void FeatureScriptCommands::CreateDeadTree(const ScriptCommandContext& ctx)
+void FeatureScriptCommands::CreateDeadTree(glm::vec3 position, const std::string&, int32_t, float, float, float, float)
 {
-	// std::cout << std::string {} + "Function " + __func__ + " not implemented.
-	// " + __FILE__ + ":" + std::to_string(__LINE__) << std::endl;
+	// spdlog::error("LHScriptX: {}:{}: Function {} not implemented.", __FILE__, __LINE__, __func__);
 }
 
-void FeatureScriptCommands::CreateScaffold(const ScriptCommandContext& ctx)
+void FeatureScriptCommands::CreateScaffold(int32_t, glm::vec3 position, int32_t, int32_t, int32_t)
 {
-	// std::cout << std::string {} + "Function " + __func__ + " not implemented.
-	// " + __FILE__ + ":" + std::to_string(__LINE__) << std::endl;
+	// spdlog::error("LHScriptX: {}:{}: Function {} not implemented.", __FILE__, __LINE__, __func__);
 }
 
-void FeatureScriptCommands::CountryChange(const ScriptCommandContext& ctx)
+void FeatureScriptCommands::CountryChange(glm::vec3 position, int32_t)
 {
-	// std::cout << std::string {} + "Function " + __func__ + " not implemented.
-	// " + __FILE__ + ":" + std::to_string(__LINE__) << std::endl;
+	// spdlog::error("LHScriptX: {}:{}: Function {} not implemented.", __FILE__, __LINE__, __func__);
 }
 
-void FeatureScriptCommands::HeightChange(const ScriptCommandContext& ctx)
+void FeatureScriptCommands::HeightChange(glm::vec3 position, int32_t)
 {
-	// std::cout << std::string {} + "Function " + __func__ + " not implemented.
-	// " + __FILE__ + ":" + std::to_string(__LINE__) << std::endl;
+	// spdlog::error("LHScriptX: {}:{}: Function {} not implemented.", __FILE__, __LINE__, __func__);
 }
 
-void FeatureScriptCommands::CreateCreature(const ScriptCommandContext& ctx)
+void FeatureScriptCommands::CreateCreature(glm::vec3 position, int32_t, int32_t)
 {
-	std::cout << std::string {} + "Function " + __func__ + " not implemented. " + __FILE__ + ":" + std::to_string(__LINE__)
-	          << std::endl;
+	spdlog::error("LHScriptX: {}:{}: Function {} not implemented.", __FILE__, __LINE__, __func__);
 }
 
-void FeatureScriptCommands::CreateCreatureFromFile(const ScriptCommandContext& ctx)
+void FeatureScriptCommands::CreateCreatureFromFile(const std::string&, int32_t, glm::vec3, glm::vec3)
```

Comment:
```suggestion
void FeatureScriptCommands::CreateCreatureFromFile(const std::string& playerName, int32_t creatureType, const std::string& creatureMind, glm::vec3 position)
```

Reply:
I would expect this to have a file path. Are you sure about these params?

Reply:
This is what I'm seeing the script files: ```CREATE_CREATURE_FROM_FILE("PLAYER_TWO", 6, "ComputerControlledCreature", "2557.58,1854.26")```

Reply:
Game prepends ```creatureMind``` name with ```"Black & White\Scripts\CreatureMind\"```